### PR TITLE
Corrected prometheus.yml configuration in Getting Started section

### PR DESCRIPTION
**Pull request by amar-chand**
Corrected indentation for static_configs and its child fields

**What this PR does / why we need it**: The config file for Prometheus was not correct in Getting Started docs.

**Which issue(s) this PR fixes**: This PR fixes the Prometheus config.

Fixes #

**Special notes for your reviewer**: Please let me know if something has to be done from my end. Thanks.

### CODE REVIEW

File: docs/sources/getting-started/getting-started-prometheus.md [hunk 1]
```diff
@@ -42,8 +42,8 @@ Prometheus node_exporter is a widely used tool that exposes system metrics. Inst
         # metrics_path defaults to '/metrics'
         # scheme defaults to 'http'.
 
-        static_configs:
-        - targets: ['<hostname>:9100']
+          static_configs:
+          - targets: ['<hostname>:9100']
```
Comment (gguillotte-grafana) [inline docs/sources/getting-started/getting-started-prometheus.md:46]:
```suggestion
               - targets: ['<hostname>:9100']
```

I think per the [Prometheus docs](https://github.com/prometheus/prometheus/blob/release-2.34/config/testdata/conf.good.yml#L51-L74), `targets` should be further indented as a child parameter of `static_configs`. This suggestion should apply this change with a 4-space indent consistent with the rest of the example.

**Review by gguillotte-grafana — CHANGES_REQUESTED** [review]
Thank you for the contribution! I think `targets` also needs to be indented one level further per the Prometheus docs. Can you please add that indent, and we can then approve and merge this fix?

File: docs/sources/getting-started/getting-started-prometheus.md (hunk 1 above)
Comment (amar-chand) [reply docs/sources/getting-started/getting-started-prometheus.md]: I have made the suggested changes. Please check and let me know if there is any other changes need to be made.

**Comment by amar-chand** [comment]
@gguillotte-grafana please re-check, I've done the needed changes

**Comment by amar-chand** [comment]
@gguillotte-grafana just a gentle reminder. Please check.

**Comment by gguillotte-grafana** [comment]
Resolving conflicts and merging.

**Pull request closed without merging**